### PR TITLE
Added example for :class:`Sector`

### PR DESCRIPTION
**Pull request by jojo96**
## Changelog / Overview

Created new example for sector class

## Motivation

More examples make it easier for new users
## Explanation for Changes

Created new example for sector class as there were no previous examples

[Documentation Reference](https://manimce--####.org.readthedocs.build)

## Checklist
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [x] I have written a descriptive PR title (see top of PR template for examples)
- [x] I have written a changelog entry for the PR or deem it unnecessary
- [ ] My new functions/classes either have a docstring or are private
- [ ] My new functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [x] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [ ] The PR is labeled correctly
- [ ] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

**Comment by kolibril13** [comment]
Great example!
Reference here: https://manimce--1647.org.readthedocs.build/en/1647/reference/manim.mobject.geometry.Sector.html?highlight=sector#manim.mobject.geometry.Sector
As it is a last frame, the self.play calls are not needed.

**Comment by jojo96** [comment]
> Great example!
> Reference here: https://manimce--1647.org.readthedocs.build/en/1647/reference/manim.mobject.geometry.Sector.html?highlight=sector#manim.mobject.geometry.Sector
> As it is a last frame, the self.play calls are not needed.

@kolibril13 so, should I just remove all the self.play calls? Then it will be a color changing animation only.

**Comment by kolibril13** [comment]
I think for this example, it is enough to only have the image and no animation at all.

**Comment by jojo96** [comment]
> I think for this example, it is enough to only have the image and no animation at all.

Changes done @kolibril13

File: manim/mobject/geometry.py [hunk 1]
```diff
@@ -764,6 +764,33 @@ def generate_points(self):
...
+
+    .. manim:: ExampleSector
+        :save_last_frame:
+
+        class ExampleSector(Scene):
+            def construct(self):
+                sector = Sector(outer_radius=2, inner_radius=1)
+                sector2 = Sector(outer_radius=2.5, inner_radius=0.8).move_to([-3, 0, 0])
+                sector.set_color(RED)
+                sector2.set_color(PINK)
```
Comment (kolibril13) [inline manim/mobject/geometry.py:780]: not necessary, as later `sector2.set_color(WHITE)` 
Also, please only use self.add once with all mobjects.
Further, self.wait() is not needed

**Comment by kolibril13** [comment]
Thanks, I left another review.

File: manim/mobject/geometry.py [hunk 2]
```diff
@@ -764,6 +764,23 @@ def generate_points(self):
...
+                self.add(sector,sector2)
```
Comment (behackl) [inline manim/mobject/geometry.py:781]:
```suggestion
                self.add(sector, sector2)
```

**Review by behackl — APPROVED** [review]
One small formatting issue, but looks good to me other than that. I've added a suggestion.

Thank you for your contribution!

**Comment by jojo96** [comment]
> One small formatting issue, but looks good to me other than that. I've added a suggestion.
> 
> Thank you for your contribution!

@behackl committed the suggestions

**Pull request merged**